app/CocoaPods/Base.lproj/MainMenu.xib

```diff
@@ -82,19 +86,19 @@
                     <modifierMask key="keyEquivalentModifierMask"/>
                     <menu key="submenu" title="File" id="bib-Uj-vzu">
                         <items>
-                            <menuItem title="Open…" keyEquivalent="o" id="IAo-SY-fd9">
+                            <menuItem title="Open Podfile…" keyEquivalent="o" id="IAo-SY-fd9">
```

Comment:
This has already been done in https://github.com/CocoaPods/CocoaPods-app/pull/216 - it's probably best you rebase this branch